LeezeIt/LeezeIt/TextFieldMaxLengths.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  TextFieldMaxLengths.swift
+//  LeezeIt
+//
+//  Created by harris ali on 8/30/22.
+//
+
+import Foundation
+import UIKit
+
+private var maxLengths = [UITextField:Int]()
```

Comment:
Is there a better approach than this? maybe making it static or something

Reply:
didn't make it generic because this is only getting used in same file.